shared/Utils.ts

```diff
@@ -53,4 +56,22 @@ export const parseJSON = (input: any) => {
 
 export const isNetworkError = (error: string) => {
   return error.includes(NETWORK_REQUEST_FAILED);
-};
\ No newline at end of file
+};
+
+export async function verifyCredentialData(
```

Comment:
instead of this Utils, move it to Utils under OpenIdVCI.

Reply:
Done